### PR TITLE
Question badge

### PR DESCRIPTION
**Pull request by jlevy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,13 @@ A lot of information on AWS is already written. Most people learn AWS by reading
...
 
-**July 2016: This is an early in-progress draft!** It’s our first attempt at assembling this information, so is certain to have omissions and errors. [**Please contribute**](CONTRIBUTING.md) by filing issues or PRs to comment, expand, correct, or otherwise improve it. This guide is *open to contributions*, so unlike a blog, it can keep improving. Like any open source effort, we combine efforts but also review ensure high quality.
+**This is an early in-progress draft!** It’s our first attempt at assembling this information, so is far from comprehensive still, and likely to have omissions or errors.
+
+[![Ask a Question](https://img.shields.io/badge/%3f-Ask%20a%20Question-ff69b4.svg)](https://airtable.com/shrXZ61VrovWfXYBg)
+
+If you have a question that is not answered here, [fill in this form](https://img.shields.io/badge/%3f-Ask%20a%20Question-ff69b4.svg), and we’ll work to include it.
+
+[**Please contribute**](CONTRIBUTING.md) by filing issues or PRs to comment, expand, correct, or otherwise improve it. This guide is *open to contributions*, so unlike a blog, it can keep improving. Like any open source effort, we combine efforts but also review ensure high quality.
```
Comment (ThanosBaskous) [inline README.md]: "but also review to ensure high quality" :)

**Comment by jlevy** [comment]
@ThanosBaskous review?

**Comment by ThanosBaskous** [comment]
One missed word, but besides that 👍 !

**Comment by jlevy** [comment]
Haha. Never think you don't need a code review. ;) Thanks!

**Pull request merged**